releases/release-5.3.4.md

```diff
@@ -0,0 +1,153 @@
+    - (dup) Fix the issue that database-level privileges are incorrectly cleaned up [#38363](https://github.com/pingcap/tidb/issues/38363)
+    - (dup) Fix the issue that the `grantor` field is missing in the `mysql.tables_priv` table [#38293](https://github.com/pingcap/tidb/issues/38293)
+    - (dup) Fix the issue that `KILL TIDB` cannot take effect immediately on idle connections [#24031](https://github.com/pingcap/tidb/issues/24031)
+    - Fix return type to string for `adddate` and `subdate` functions [#36394](https://github.com/pingcap/tidb/issues/36394)
+    - Fix restore table option `INSERT_METHOD` incompatible with MySQL [#38368](https://github.com/pingcap/tidb/issues/38368)
+    - Fix authentication with MySQL 5.1 and older clients [#29725](https://github.com/pingcap/tidb/issues/29725)
+
+    <!--executor owner: @zanmato1984-->
+
+    - (dup) Fix wrong results of `GREATEST` and `LEAST` when passing in unsigned `BIGINT` arguments [#30101](https://github.com/pingcap/tidb/issues/30101)
+    - (dup) Fix the issue that the result of `concat(ifnull(time(3))` in TiDB is different from that in MySQL [#29498](https://github.com/pingcap/tidb/issues/29498)
+    - avoid sum from avg overflow [#29952](https://github.com/pingcap/tidb/issues/29952)
+    - add an unit test case for unreasonable invoking Close [#30587](https://github.com/pingcap/tidb/issues/27125)
+    - HashJoinExec checks the buildError even if the probeSide is empty [#30289](https://github.com/pingcap/tidb/issues/30289)
+    - expression: resize the result for IfXXSig [#37414](https://github.com/pingcap/tidb/issues/37414)
+    - change date_add and date_sub string_(int/string/real/decimal) function return type to string. [#27573](https://github.com/pingcap/tidb/issues/27573)
+    - fix hashjoin goleak [#39026](https://github.com/pingcap/tidb/issues/39026)
+    - fix: the results of tikv and tiflash are different [#37258](https://github.com/pingcap/tidb/issues/37258)
+
+    <!--transaction owner: @cfzjywxk-->
+    
+    - Fix the issue explain analyze with DML executors may respond to the client before the transaction commit has finished. [#37373](https://github.com/pingcap/tidb/issues/37373)
+
+    <!--planner owner: @qw4990-->
+
+    - (dup) Fix the issue that the EXECUTE might throw an unexpected error in specific scenarios [#37187](https://github.com/pingcap/tidb/issues/37187)
+    - (dup) Fix the issue that `GROUP CONCAT` with `ORDER BY` might fail when the `ORDER BY` clause contains a correlated subquery [#18216](https://github.com/pingcap/tidb/issues/18216)
+    - Fix the issue that set wrong length and width for Decimal and Real when using plan-cache [#29565](https://github.com/pingcap/tidb/issues/29565)
+    - add an unit test case for unreasonable invoking Close [#27125](https://github.com/pingcap/tidb/issues/27125)
```

Comment:
```suggestion
    - Fix the issue that the EXECUTE might throw an unexpected error in specific scenarios [#37187](https://github.com/pingcap/tidb/issues/37187)
    - Fix the issue that `GROUP CONCAT` with `ORDER BY` might fail when the `ORDER BY` clause contains a correlated subquery [#18216](https://github.com/pingcap/tidb/issues/18216)
    - Fix the issue that wrong length and width are set for Decimal and Real when using plan-cache [#29565](https://github.com/pingcap/tidb/issues/29565)
```

Reply:
@chrysan The `dup` label marks the notes that have been released in other patch or minor releases, such as v6.1.2. I will remove the label after final review. For now, we can keep it and reduce the workload for both tech reviewers and editorial reviewers.